### ISSUE TITLE
Change to a different layout from keyboard and/or command line actions.

### ISSUE THREAD

**Issue by ZhichaoHong** [RamonUnch/AltSnap#479]
Now AltSnap supports multiple layouts, it will be helpful to switch to a different layout. For example, when using [FuPeiJiang/VD.ahk at 273edefde9b790ddeee7c0a306af65fdc93fb5b5](https://github.com/FuPeiJiang/VD.ahk/tree/273edefde9b790ddeee7c0a306af65fdc93fb5b5#readme), it can register other commands when the virtual desktop changes. It can convenient to switch to different layout when virtual desktop changed.

**Comment by RamonUnch** [comment]
Good idea indeed, Implementation of command line option is trivial and would be the most convenient for you:
[AltSnap1.62test9_i386.zip](https://github.com/RamonUnch/AltSnap/files/12910623/AltSnap1.62test9_i386.zip)
[AltSnap1.62test9_x64.zip](https://github.com/RamonUnch/AltSnap/files/12910624/AltSnap1.62test9_x64.zip)

**Comment by ZhichaoHong** [comment]
Worked great! Thanks for the quick addition, @RamonUnch!
